project_generation/helpers.go

```diff
@@ -400,10 +400,15 @@ func InitGoModules(ctx context.Context, pathToRepo, name string) error {
 
 // FinaliseModules will run go build ./... to generate go modules dependency management files
 func FinaliseModules(ctx context.Context, pathToRepo string) {
-	cmd := exec.Command("go", "build", "./...")
-	cmd.Dir = pathToRepo
-	err := cmd.Run()
-	if err != nil {
+	build := exec.Command("go", "build", "./...")
```

Comment:
I wonder whether doing the build step after the format step is better, but I doubt it's crucial.
Also thinking about it, it may be better to duplicate the function rather than adding a step to it as this function is specifically concerned with building the `go.mod` and `go.sum` files. I would be tempted to find the locations where `FinaliseModules(…)` is called and call `FormatGoSource(…)` as well.